tools/dev-server.js

```diff
@@ -202,5 +201,6 @@ function printWelcomeMessage(args) {
     borderStyle: "bold",
     backgroundColor: "black",
   };
+  const { default: boxen } = await import("boxen");
```

Comment:
Current boxen also has dual module support?

Reply:
No idea... seems to just want this.